lang/de.xml

```diff
@@ -247,7 +247,8 @@ Du kannst Trophäen in Anwendungen verdienen, die die Trophäen-Funktionen unter
 		<delete_user>Lösche Profil</delete_user>
 		<user_delete>Wähle das Profil, das gelöscht werden soll.</user_delete>
 		<user_delete_msg>Folgendes Profil wird gelöscht.</user_delete_msg>
-		<user_delete_message>Wenn du das Profil löscht, werden alle zugehörigen Nutzerdaten und Trophäen gelöscht.</user_delete_message>
+		<user_delete_message>Wenn du das Profil löscht, werden alle zugehörigen Nutzerdaten
```

Comment:
same here, no using return line, using all time textwrapped for this case

Reply:
> same here, no using return line, using all time textwrapped for this case

finished.